cmd/containerd/command/main_windows.go

```diff
@@ -24,7 +24,9 @@ import (
 	"unsafe"
 
 	winio "github.com/Microsoft/go-winio"
+	"github.com/Microsoft/go-winio/pkg/etw"
 	"github.com/Microsoft/go-winio/pkg/etwlogrus"
+	"github.com/Microsoft/go-winio/pkg/guid"
```

Comment:
just when we got rid of some UUID implementations, a new one is added 😂  (sorry, couldn't resist)